docs/tutorials/intro_tutorial.rst

```diff
@@ -48,27 +48,42 @@ Let's get started.
 Installation
 ~~~~~~~~~~~~
 
-To start, install Mesa. We recommend doing this in a `virtual
-environment <https://virtualenvwrapper.readthedocs.org/en/stable/>`__,
-but make sure your environment is set up with Python 3. Mesa requires
-Python3 and does not work in Python 2 environments.
+To start, install Mesa. We recommend using a `pipenv <https://docs.pipenv.org/en/latest/>`_,
+which combines the `virtual
+environment <https://virtualenvwrapper.readthedocs.org/en/stable/>`_
+along with the `dotenv <https://github.com/theskumar/python-dotenv>`_ projects
+simplifying your experience, but make sure your environment is set up with Python 3.
+Mesa requires Python3 and does not work in Python 2 environments.
 
-To install Mesa, simply:
+To install Mesa, with pipenv simply clone the repository and run:
```

Comment:
I think this is way too complicated for a intro tutorial. This sentence assumes new users already have pipenv and git installed and know how to/are willing to clone the repository. I think it is fine to recommend pipenv (although I still don't get why we need a pipfile and not just recommend `pipenv install mesa`, see #678 ), but would leave the instructions to install with pip.